docker/Dockerfile

```diff
@@ -203,8 +201,21 @@ USER root
 # the user connects with the postgres user, the DuckDB folder will be created in the /var/lib/postgresql directory. We
 # chmod both directories to ensure that DuckDB can write to them no matter which user is used to connect to the database.
 RUN mkdir .duckdb/ && chmod -R a+rwX .duckdb/ && \
-    mkdir /var/lib/postgresql/.duckdb/ && chmod -R a+rwX /var/lib/postgresql/.duckdb/ && \
-    apt-get install -y --no-install-recommends libpq5 && rm -rf /var/lib/apt/lists/*
+    mkdir /var/lib/postgresql/.duckdb/ && \
+    chmod -R a+rwX /var/lib/postgresql/.duckdb/
+
+# Installs Barman Cloud and its dependencies for Azure, Google, and AWS via NPM and cleans up the installation.
```

Comment:
This is installing via `pip` tho, not NPM right?

Reply:
OMG! What was I thinking!?